dotnet/src/Connectors/Connectors.Memory.Redis/RedisVectorDistanceMetrics.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.SemanticKernel.Connectors.Memory.Redis;
+
+/// <summary>
+/// Supported distance metrics are {L2, IP, COSINE}. The default value is "COSINE".
+/// <see href="https://redis.io/docs/interact/search-and-query/search/vectors/"/>
+/// </summary>
+public enum VectorDistanceMetrics
```

Comment:
DO use a singular type name for an enumeration unless its values are bit fields.
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations

Reply:
Good catch, resolved.